projects/plugins/jetpack/modules/memberships/class-jetpack-memberships.php

```diff
@@ -747,6 +748,19 @@ public static function is_enabled_jetpack_recurring_payments() {
 		return $api_available;
 	}
 
+	/**
+	 * Whether to enable the blocks in the editor.
+	 * All Monetize blocks (except Simple Payments) need an active connecting and a user with at least `edit_posts` capability
+	 *
+	 * @return bool
+	 */
+	public static function should_enable_monetize_blocks_in_editor() {
+		$jetpack_ready_and_user_connected = Jetpack::is_connection_ready() &&
```

Comment:
Since we're bring the connection manager here, maybe we could make things cleaner by calling the `is_connected` method from the Manager class here, instead of the big `Jetpack` class?

Reply:
I pushed some change in this way.